content/SCALETutorials/Datasets/PermissionsSCALE.md

```diff
@@ -82,6 +82,16 @@ Click **Save** now if you do not want to [use an ACL preset](#adding-a-new-prese
 
 See [Edit ACL Screen]({{< ref "EditACLScreens" >}}) for information on the ACL editor screens and setting options.
 
+{{< hint type=important >}}
+Users can grant `root` permissions to containers and instances through an unprivileged root account using the ACL editor in the UI or the TrueNAS API. 
+
+To ensure functionality, add an ACE for the `truenas_container_unpriv_root` user and assign the appropriate permissions (such as **Read**, **Modify**, and **Execute**).
```

Comment:
incorrect use of emphasis here as well, and is this what they see on the screen? if it is use bold, if not, add the name they see on the screen and use bold. The permissions don't need to be in bold here as you are not really talking about the UI screen elements just the permission levels.
```suggestion
To ensure functionality, add an ACE for the truenas_container_unpriv_root user and assign the appropriate permissions (such as read, modify, and execute).
```